src/components/Checkbox/Checkbox.st.css

```diff
@@ -16,31 +16,33 @@
 :vars {
     /*Defaults*/
     DefaultTextColor: color-5;
-    DefaultBorderColor: color-5;
+    DefaultBoxBorderColor: color-5;
     DefaultIconColor: color-8;
+    DefaultBoxColor: color-1;
     DisabledColor: color-3;
     Font: font("{theme: 'Body-M', size: '16px', lineHeight: '24px'}");
 }
 
 :vars {
     /*Overrides*/
     TextColor: --overridable;
-    BorderColor: --overridable;
```

Comment:
We can't remove or rename existing variables, because consumers might already be using them, so this might break their projects.
I agree that the naming here is confusing and bad, but we'll have to leave it like that

Reply:
Is it OK if I bring back the overridable BorderColor but keep it off the storybook? I think it should be deprecated.

Reply:
For the moment it's better to leave it in the storybook.
I'll raise this issue in our team, and we have our deprecation process for this.
But for now, just leave the api as it was + add the addition api for the box theme